pkg/api/rest/OWNERS

```diff
@@ -3,3 +3,4 @@ reviewers:
 - wojtek-t
 - thockin
 - caesarxuchao
+- smarterclayton
```

Comment:
Part of the the reason for this was to catch the original authors, which this looks like it has done for a number of important core packages that have been around for longer than 1 year.